src/common/components/UserProfileAvatar.tsx

```diff
@@ -0,0 +1,20 @@
+import { Avatar, type AvatarProps } from "@radix-ui/themes";
+
+type UserProfileAvatarProps = Omit<
+  AvatarProps,
+  "radius" | "variant" | "color" | "fallback"
+>;
+
+const UserProfileAvatar = ({ ...props }: UserProfileAvatarProps) => {
```

Comment:
여긴 다른 prop이 없어서 이렇게 해도 괜찮을 것 같네용!
개인 취향이니 참고만 해주셔요!

```suggestion
const UserProfileAvatar = (props: UserProfileAvatarProps) => {
```

Reply:
오 그렇네용 불필요한 스프레드인것 같아 [반영했어요](https://github.com/Mo-yoy/Moyo-Frontend/pull/15/commits/8d6e3bf7684951c08111d9631a95fc4c169edfa6) !! 감사합니댜